packages/dds/tree/api-report/tree.api.md

```diff
@@ -2024,18 +2029,25 @@ export type TreeValue<TSchema extends ValueSchema = ValueSchema> = [
 
 // @public
 export interface TreeView<TSchema extends ImplicitFieldSchema> extends IDisposable {
-    readonly error?: SchemaIncompatible;
+    readonly compatibility: SchemaCompatibilityStatus;
     readonly events: ISubscribable<TreeViewEvents>;
+    initialize(content: InsertableTreeFieldFromImplicitField<TSchema>): void;
     get root(): TreeFieldFromImplicitField<TSchema>;
     set root(newRoot: InsertableTreeFieldFromImplicitField<TSchema>);
     upgradeSchema(): void;
 }
 
+// @public
+export interface TreeViewConfiguration<TSchema extends ImplicitFieldSchema = ImplicitFieldSchema> {
+    schema: TSchema;
```

Comment:
This being mutable seems like it might allow things we do not want. I suspect this interface is intended to be covariant over the schema, so this should probably be read-only and maybe mark the interface with "in" explicitly.

If you do want this to be invariant them mutability is fine, but I don't think we should have a default type argument in that case as that is mainly used when the default is such all versions of the interface are compatible with the version without a parameter (which this would do if covariant/readonly)